qemu/tests/zero_copy.py

```diff
@@ -52,6 +52,9 @@ def enable_zerocopytx_in_host(enable=True):
         enable_zerocopytx_in_host(False)
 
     error.context("Boot vm with 'vhost=on'", logging.info)
+    if params.get("nettype") == "user":
+        raise error.TestNAError("Unable start test with user networking,"
```

Comment:
@huangwei let me clarify this, as the `autotest` package is going to be deprecated by `avocado-vt`, new codes should avoid to use any functionality provided by it. For old codes, since those functionalities are supported till now, it's okay to replace them once sending the new change.